libpimeval/src/pimPerfEnergyModels.cpp

```diff
@@ -94,7 +95,7 @@ pimParamsPerf::getPerfEnergyBitSerial(PimDeviceEnum deviceType, PimCmdEnum cmdTy
         if (it3 != it2->second.end()) {
           const auto& [numR, numW, numL] = it3->second;
           msRuntime += m_tR * numR + m_tW * numW + m_tL * numL;
-          mjEnergy += (m_eL * numL * obj.getMaxElementsPerRegion() * obj.getBitsPerElement()) * obj.getNumCoresUsed() + (m_eAP * numR + m_eAP * numW) * m_numChipsPerRank * numRanks;
+          mjEnergy += ((m_eL * numL * obj.getMaxElementsPerRegion()) + (m_eAP * numR + m_eAP * numW)) * numCores;
```

Comment:
Just to confirm that "(m_eL * numL * obj.getMaxElementsPerRegion()" should also be multiplied with numCore, right?

Reply:
Yes. Fixing this

Reply:
Ok, it's been multiplied.